### PR TITLE
Testing - Packaging v2

### PR DESCRIPTION
**Pull request by Salamandar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Salamandar** [comment]
!testme

File: scripts/_common.sh [hunk 1]
```diff
@@ -133,7 +120,7 @@ import_template () {
 	then
 		sid=$(curl $curlOptions \
```
Comment (alexAubin) [inline scripts/_common.sh:121]: This curl call and the previous one probably don't work anymore in private install

I'm not sure how it was supposed to work before, but anyway, ideally one should rely on ynh_local_curl which will temporarily make the app public : https://github.com/YunoHost/yunohost/blob/dev/helpers/utils#L435

**Review by alexAubin — CHANGES_REQUESTED** [review]

File: manifest.toml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+    [install.path]
+    type = "path"
+    default = "/zabbix"
+
+    [install.init_main_permission]
+    help.en = "A public app doesn't need SSO auth : the auth page is opened for everyone"
+    help.fr = "Une application publique ne nécessite pas une authentification SSO : sa page d'authentication est ouverte à tous"
+    type = "group"
+    default = false
```
Comment (Thovi98) [inline manifest.toml:43]:
```suggestion
    default = "visitors"
```
 `false` is not a valid value for Bookworm

**Comment by alexAubin** [comment]
So TL;DR the status is : 
- this PR is working-ish on bullseye, but the private install probably is broken because there's a raw curl which doesn't properly temporarily enable visitors on the endpoint
- it doesn't work on bookworm because zabbix 5.x depends on libssl1.1 which doesn't exist anymore on bookworm. We should upgrade to 6.x probably, but as shown in https://github.com/YunoHost-Apps/zabbix_ynh/pull/46, the initial provisioning of the mysql DB changes (?) and probably a migration need to be performed ?

**Comment by alexAubin** [comment]
Hmokay, I managed to fix the curl issue, let's release this and iterate for bookworm

**Pull request merged**